.github/workflows/pr-auto-merge.yml

```diff
@@ -0,0 +1,26 @@
+name: auto-merge flagged PR
+on:
+  pull_request_target:
+    types: ['labeled']
+
+permissions:
+  pull-requests: write
+  contents: write
+
+## This job will auto merge PR that has been labeled with 'auto-merge'
+jobs:
+  activate-auto-merge:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'auto-merge' }}
+    steps:
+      - name: Approve a PR
+        run: gh pr review --approve "$PR_URL"
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: ${{secrets.WEBTEAM_AUTOMERGE_TOKEN}}
+
+      - name: Enable auto-merge for Dependabot PRs
```

Comment:
you think renaming that part too would be useful or no?

Reply:
damn 👍